src/main/java/redis/clients/jedis/Connection.java

```diff
@@ -370,6 +370,23 @@ public List<Object> getMany(final int count) {
     return responses;
   }
 
+  /**
+   * Check if the client name libname, libver, characters are legal
+   * @param info the name
+   * @return Returns true if legal, false throws exception
+   * @throws JedisException if characters illegal
+   */
+  public static boolean validateClientInfo(String info) {
```

Comment:
It should be private.
If you really want it to be public, util package is a better place.

Reply:
I will change it to public.
I currently haven't found a suitable class in the util package to store it (I should need to create a new class, but I haven't come up with a good name). If you prefer to move it to util, I can also accept it.

Reply:
> but I haven't come up with a good name

I understand :smile: